REPLICATION_POLICIES.md

```diff
@@ -0,0 +1,292 @@
+Replication policies work quite differently on Android from Java. Please see the relevant section of this guide 
+for [Android](#android-replication-policies) or [Java](#java-replication-policies).
+
+# Android replication policies
+Replication policies on Android run in a `Service` to allow them to run independently of your main application and to allow
+them to be restarted if they are killed by the operating system.  To use replication policies, you need to create a subclass
```

Comment:
Minor grammar issue here
>you need to create a subclass one of

I think needs to be either:
* you need to create a subclass of one of
* you need to subclass one of